File: array/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: list[list[int]]) -> int:
        maximumWealthSoFar = 0

        for customer in accounts:
            currentCustomerWealth = 0

            for bank in customer:
                currentCustomerWealth += bank
            
            maximumWealthSoFar = max(maximumWealthSoFar, currentCustomerWealth)

        return maximumWealthSoFar

File: array/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
      int maximumWealthSoFar = 0;

      for (int[] customer: accounts) {
        int currentCustomerWealth = 0;

        for (int bank: customer) {
          currentCustomerWealth += bank;
        }

        maximumWealthSoFar = Math.max(maximumWealthSoFar, currentCustomerWealth);
      }

      return maximumWealthSoFar;
    }
}
